undertow/src/test/resources/org/wildfly/extension/undertow/undertow-3.0.xml

```diff
@@ -27,11 +27,11 @@
     <buffer-cache name="extra" buffer-size="1025" buffers-per-region="1054" max-regions="15"/>
 
     <server name="some-server" default-host="other-host" servlet-container="myContainer">
-        <ajp-listener name="ajp-connector" socket-binding="ajp" redirect-socket="ajps" max-parameters="5000" tcp-keep-alive="true" tcp-backlog="500" receive-buffer="5000" send-buffer="50000" resolve-peer-address="true" no-request-timeout="10000" request-parse-timeout="2000"/>
+        <ajp-listener name="ajp-connector" disallowed-methods="FOO" socket-binding="ajp" redirect-socket="ajps" max-parameters="5000" tcp-keep-alive="true" tcp-backlog="500" receive-buffer="5000" send-buffer="50000" resolve-peer-address="true" no-request-timeout="10000" request-parse-timeout="2000"/>
```

Comment:
probably good idea to also test more than one, so disallowed-methods="FOO BAR" should disallow two methods.